DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -75,6 +72,16 @@ public void GenerateAndSendPasswordResetLink(string emailAddress, string baseUrl
             emailService.SendEmail(resetPasswordEmail);

```

Comment:
I think FirstName can technically be null in the db, so we should handle it in GeneratePasswordResetEmail :)